tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/JSDesugaring.scala

```diff
@@ -247,10 +245,10 @@ private[emitter] class JSDesugaring(semantics: Semantics,
       desugar.transformExpr(tree)(env)
   }
 
-  private[emitter] implicit def transformIdent(ident: Ident): js.Ident =
+  implicit def transformIdent(ident: Ident): js.Ident =
     js.Ident(ident.name, ident.originalName)(ident.pos)
```

Comment:
Why is this not private? If it needs to be public, I feel like it should be defined in `JSGen`.